samples/matter/common/src/diagnostic/diagnostic_logs_crash.cpp

```diff
@@ -190,16 +190,20 @@ CHIP_ERROR CrashData::GetLogs(chip::MutableByteSpan &outBuffer, bool &outIsEndOf
 	return err;
 }
 
-CHIP_ERROR CrashData::FinishLogs()
+CHIP_ERROR DiagnosticLogsCrash::FinishLogs()
 {
 #ifdef CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REMOVE_CRASH_AFTER_READ
-	int ret = retention_clear(kCrashDataMem);
-	VerifyOrReturnError(0 == ret, System::MapErrorZephyr(ret));
+	return Clear();
 #endif /* CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS_REMOVE_CRASH_AFTER_READ */
 
 	return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR DiagnosticLogsCrash::Clear()
+{
+	return System::MapErrorZephyr(retention_clear(kCrashDataMem));
```

Comment:
Should we check if `device_is_ready(kCrashDataMem)` here and in other places?

Reply:
Hmm to be honest, I don't know. I don't see anyone doing it, even in the Zephyr's example.